src/lib.rs

```diff
@@ -745,4 +746,25 @@ mod wpt {
         );
         assert!(messages.next().is_none());
     }
+
+    #[test]
+    fn decode_stream_when_fed_by_line() {
```

Comment:
Could you move this test to the other block? Under `mod tests {}` a bit further up in this file. The `mod wpt {}` block is intended for tests ported from the [web-platform-tests](https://github.com/web-platform-tests/wpt/tree/master/eventsource) suite.